src/pages/Profile/Profile.tsx

```diff
@@ -63,21 +77,29 @@ const Profile: FC = () => {
     )
   }
 
+  function renderEditButton() {
+    if (isCurrentUser)
+      return <Chip className={classes.editButton} color="primary" label="Edit Info" />
+  }
```

Comment:
![Screenshot 2020-05-23 at 13 10 03](https://user-images.githubusercontent.com/42270585/82728047-bde4d580-9cf6-11ea-83cf-cd6502376ad3.png)
```suggestion
  function renderEditButton() {
    if (!isCurrentUser) return null
    
    return <Chip className={classes.editButton} color="primary" label="Edit Info" />
  }
```